### ISSUE TITLE
Allow back split mode, missing in v0.177.0

### ISSUE THREAD

**Issue by turuto** [MeoMix/StreamusChromeExtension#592]
Hello.
First of all I wat to make clear that I reckon Streamus one of the best and most useful pieces of software that I've been using lately.
With the v.0177.0 update we have lost the split mode that showed simultaneously the playlists and the stream.
It seems to me that we have lost the ability of easily sorting and organizing the songs in both lists, and taking in and out songs from the stream to the playlist.
Maybe I miss this feature we had as I think the most of my playlists not as a collection of Songs to be played in order but merely as folders with songs sorted alphabetically, (think of concerts, Soundtracks, etc.). So my use of streamus was based on working with the two list at the same time.
I think that allowing the split view in the Streamus settings would be very useful for people that make use of playlists as i do (don't know if there are many of us or if I'm the only one, though).
Hope I managed to explain well. Thank a lot for your work

**Comment by Derpp** [comment]
YouTube asked (demanded) that the video be shown. For this to work, Sean had to make the the extension use the maximum amount of space allowed by Chrome. This means no split pane, at least for now, to keep things clean I'm guessing. 

Perhaps we will get the split pane back in a future release.

**Comment by MeoMix** [comment]
Yeah, the split pane will be coming back fa sho. The intent was that the video would be toggled on/off and take up the full view, but since I'm not allowed to toggle it on/off the single pane view lost some of it's use and isn't as well received as I thought.

Going to try and write it so that it's an option to have either a single pane or a split pane though.

**Comment by turuto** [comment]
Yes, i understand that the video needs to be shown. That's a fact. But what i don't know if is any specs or requirements about the width of it. I will try to make a screenshot of how could we accommodate the video leaving room for the split, just if it could be of any help

**Comment by turuto** [comment]
Yes, i understand that the video needs to be shown. That's a fact. But what i don't know if is any specs or requirements about the width of it. I will try to make a screenshot of how could we accommodate the video leaving room for the split, just if it could be of any help

**Comment by MeoMix** [comment]
https://developers.google.com/youtube/iframe_api_reference#Requirements

"Embedded players must have a viewport that is at least 200px by 200px." I chose to do 200px tall with 16:9 dimensions so it could be less wide, but it'd look kind of gross.

Feel free to mock up screenshots though. I'd love to have more ideas/help brainstorming on how to improve the UI. :)

**Comment by turuto** [comment]
Though I have not access to neither my computer neither my chrome, I've managed to grab a screenshot of Streamus from Google and made up some screenshot of how we could accommodate the split view with the video.
![streamus](https://cloud.githubusercontent.com/assets/9445216/8483675/9b68c320-20f2-11e5-9377-f7055c2e7bba.jpg)

In this version I've pointed with red arrows that the border collumns could be draggable so we coud adjust the collumn width and  hence making the video bigger or smaller acordding to our needs.
![streamus_draggable](https://cloud.githubusercontent.com/assets/9445216/8483713/e24ccbc4-20f2-11e5-850f-77254775ba9e.jpg)


Obviuosly, if we drag the left border to the left, the stream would get wider,but shorter as the video would get wider and taller too.
